contrib/auto-render/auto-render.js

```diff
@@ -48,12 +49,14 @@ const renderMathInText = function(text, delimiters) {
     return fragment;
 };
 
-const renderElem = function(elem, delimiters, ignoredTags) {
+const renderElem = function(elem, delimiters, ignoredTags, renderOptions) {
     for (let i = 0; i < elem.childNodes.length; i++) {
         const childNode = elem.childNodes[i];
         if (childNode.nodeType === 3) {
             // Text node
-            const frag = renderMathInText(childNode.textContent, delimiters);
+            const frag = renderMathInText(
+              childNode.textContent, delimiters, renderOptions
```

Comment:
Nit prick: we have 4 spaces of indentation everywhere else, so please conform to that. In some places we have the closing `)` on the same line as the last argument, but that is already inconsistent.